### PR TITLE
Rebuild for r-base 4.4 and UCRT mingw toolchain

### PR DESCRIPTION
**Pull request by danielnachun**

### CODE REVIEW

**Review by bgruening — APPROVED** [review]

File: recipe/build.sh [hunk 1]
```diff
@@ -1,36 +1,5 @@
...
-            install_name_tool -change /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libc++.1.dylib "${PREFIX}"/lib/libc++.1.dylib "${SHARED_LIB}" || true
-          done <   <(find . \( -type f -iname "*.dylib" -or -iname "*.so" -or -iname "R" \) -print0)
-        popd
-      done
-    popd
-  fi
-fi
+
+export DISABLE_AUTOBREW=1
+${R} CMD INSTALL --build .
```
Comment (mfansler) [inline recipe/build.sh]: We should always pass the arguments:

```suggestion
${R} CMD INSTALL --build . ${R_ARGS}
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -32,14 +27,13 @@ requirements:
 
 test:
   commands:
-    - $R -e "library('venneuler')"           # [not win]
-    - "\"%R%\" -e \"library('venneuler')\""  # [win]
+    - $R -e "library('venneuler')"
```
Comment (mfansler) [inline recipe/meta.yaml]: Please don't undo this. That is the current boilerplate and should be retained.

```suggestion
    - $R -e "library('venneuler')"           # [not win]
    - "\"%R%\" -e \"library('venneuler')\""  # [win]
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -32,14 +27,13 @@ requirements:
...
 
 about:
   home: http://www.rforge.net/venneuler/, http://www.cs.uic.edu/~wilkinson/
   license: MPL-1.1
+  license_file: LICENSE
   summary: Calculates and displays Venn and Euler Diagrams
-  license_family: OTHER
```
Comment (mfansler) [inline recipe/meta.yaml:42]: Please restore.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,6 +1,4 @@
 {% set version = "1.1-4" %}
-{% set posix = 'm2-' if win else '' %}
```
Comment (mfansler) [inline recipe/meta.yaml:2]: Retain.

File: recipe/meta.yaml [hunk 5]
```diff
@@ -13,16 +11,13 @@ source:
...
-  merge_build_host: true  # [win]
   number: 0
   noarch: generic
   rpaths:
     - lib/R/lib/
     - lib/
 
 requirements:
-  build:
-    - {{ posix }}zip               # [win]
```
Comment (mfansler) [inline recipe/meta.yaml]: Retain.

**Review by mfansler — CHANGES_REQUESTED** [review]

**Comment by mfansler** [comment]
Hi @danielnachun. Since you're joining the team, maybe let's be a bit more strict about keeping the standard that would come out of `conda_r_skeleton_helper` rather than cutting off what might currently seem as unnecessary. The advantage to keeping the Windows boilerplate around is that all our recipes are robust to readily convertible to non-`noarch`.

Also, in the future please always commit your changes first, then run a `conda-smithy rerender` and commit with the suggested commit message. That provides a transparent record of when rerendering was done and the exact version of `conda-forge-pinning` that was applied.

**Comment by danielnachun** [comment]
Thanks @mfansler, I wasn't sure if those `#[win]` lines were leftovers that are supposed to be removed for `noarch` packages since I was mostly fixing non-`noarch` packages until now.  I guess for new R recipe submission in conda-forge they might result in linting failures, but for existing ones it's obviously allowed.  I'll also fix up my commit messages for the rerenders.  I guess I can use the bot to rerender most of the time after commit my manual changes as I've seen you do before.  I'll make sure to double check with other R maintainers before merging anything I'm not sure about.

**Comment by danielnachun** [comment]
I resolved all the requested changes here and made the same changes to the other PRs I made for license issues as well, so I'm going to merge them now.

**Pull request merged**